src/index.html

```diff
@@ -137,11 +144,17 @@ <h1>Instant, themed Twitch chat overlays</h1>
 			if (chosenValue === "false") {
 				chosenValue = false;
 			}
-			if (chosenValue == value) {
+			if (chosenValue == defaultValue || chosenValue === '') {
```

Comment:
```suggestion
			if (chosenValue == defaultValue) {
```
Now that the default for `clearMessageAfter` is an empty string like `hideMessagesFrom`, the check that's in place should cover the case where this input isn't touched during urlbuilding.

I like the renaming to  `defaultValue`, it makes the intent clearer!

Reply:
I merged in this change, and realized a case where this `chosenValue === ''` check was needed: the `showLatestMessages` field. The default value for that is `100`, but that's now being represented as a placeholder, and not an initial value. As a result, leaving the field blank (`''`) is not going to be equal to the default (`100`). I'm adding this check back in — but thank you for the suggestion!

Reply:
Ah good catch. It changed to placeholder to keep consistency with the "Show Commands" input?
To make sure my understanding is correct: now the defaults are used in the serverless function if nothing is provided for that option in the URL builder. The defaults are only shown in the URLbuilder as guides for "this is a possible thing that goes here".

In that case, can we consolidate all default checks? Changing them to "does this input option contain anything"?
Unsure, will test by console.logging later.

With the `=== ''` check, lines 140-143 will never be hit, so those can go too.
https://github.com/BenDMyers/showmy.chat/pull/91/files#diff-ce434ef5cc161194b450135a3586eb89357fc270e8aaa8e336db5a04e8d52fbeR140-R143

Reply:
Ah, great catch! I had totally missed that this negated the need for the `if (key == 'showCommands' && chosenValue === '')` check - I can go and remove that!

Reply:
I think the question of "if someone manually puts a value that's the same as the default, should that be explicitly passed in the URL" is a good one, and one that I'm not sure I have an answer for!

Allowing folks to more explicitly set values equal to the defaults would mean that if, for some reason, we decided to change a default (a move that should definitely be done rarely, but which I don't think is entirely out of the question), those users' overlays don't change from underneath them. That's not, altogether, a bad idea.

Maybe this check is best handled as:

```suggestion
if (chosenValue === '') {
```
?

Reply:
Yep, that should catch the inputs that haven't been touched, and include everything that was explicitly set!
Let's use this.